uint16 id
float32 pos_estimate
float32 vel_estimate
float32  torque_estimate
bool   traj_done
bool   axis_error